src/scripts/ic8/hiv/hiv_filehandlers.py

```diff
@@ -728,33 +728,27 @@ def _turn_workbook_into_df(self, file: Path) -> pd.DataFrame:
             # Extract Step4 values for 2022
             step4_2022 = csv_df[(csv_df['scenario_descriptor'] == 'Step4') & (csv_df['year'] == 2022)]
 
-            # Merge Step4 values with Step13 rows
-            df_step13_updated = csv_df[mask_step13_2022].merge(
-                step4_2022[['country'] + column_names],
-                on='country',
-                suffixes=('', '_step4')
+            # Create a mapping for Step4 values (x, y, z) by country
```

Comment:
explain

Reply:
@tbhallett , I have explained that in the Peter S branch